website/src/components/articles/article-content-footer.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC } from "react";
+import styled from "styled-components";
+
+interface Props {
```

Comment:
To be consistent lets call it `ArticleContentFooterProperties`.

Reply:
Made the change, but I don't see the point in using the longer name, if we're not exporting it. Also not a big fan of `Properties` over `Props`. But if that's a convention here I'll go with it :)

Reply:
We can discuss conventions of course. `Props` is fine with me, but then consistent. We can change later.